src/draw/lv_draw.h

```diff
@@ -36,11 +35,74 @@ extern "C" {
 /**********************
  *      TYPEDEFS
  **********************/
+typedef struct _lv_draw_backend_t{
+    struct _lv_draw_backend_t * base;
+
+    void * ctx;
+
+    void (*draw_rect)(const lv_area_t * coords, const lv_area_t * clip, const lv_draw_rect_dsc_t * dsc);
```

Comment:
Please add `lv_draw_backend_t` or `ctx` to the end of parameters list. By doing so, I can access SDL renderers and offscreen textures, and stop setting non-buffer pointer to `lv_disp_draw_buf_t`, and leave `lv_disp_drv_t` untouched.

Reply:
Ok, I will.